themes/janeswalk/js/components/pages/WalkPage/WalkPage.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+
+//TODO: Define as a path in a common place to share with other components
+import ItineraryActions from '../../../../../../blocks/page_list/templates/itinerary/ItineraryActions';
+import ItineraryStore from '../../../../../../blocks/page_list/templates/itinerary/ItineraryStore';
+
+import WalkHeader from './WalkHeader.jsx';
+import WalkDescription from './WalkDescription.jsx';
+import WalkRoute from './WalkRoute.jsx';
+import WalkAccessibility from './WalkAccessibility.jsx';
+import WalkPublicTransit from './WalkPublicTransit.jsx';
+import WalkStart from './WalkStart.jsx';
+import WalkTeam from './WalkTeam.jsx';
+import WalkMenu from './WalkMenu.jsx';
+import WalkMap from './WalkMap.jsx';
+
+import {walk} from './WalkStaticData';
+import './view.less';
+
+const walkId = walk.walk.id;
+const itineraryListId = ItineraryStore.getAllLists()[0].id; //for stubbed data, assumed first list is Itinerary
+const favoriteListId = ItineraryStore.getAllLists()[1].id; //for stubbed data, assumed second list is fav
```

Comment:
so two separate calls: getItineraryLists() and getFavouriteLists() would make sense